packages/data-context/src/data/ProjectLifecycleManager.ts

```diff
@@ -582,6 +583,24 @@ export class ProjectLifecycleManager {
 
       throw getError('CONFIG_VALIDATION_ERROR', 'configFile', file || null, errMsg)
     })
+
+    return this.ctx._apis.configApi.validateLaunchpadConfigBreakingChanges(
+      config,
+      (type, obj) => {
```

Comment:
Can we pull in the types: https://github.com/cypress-io/cypress/blob/083695857dbc4dc3ad7f991cf29967b06523efad/packages/config/lib/validation.d.ts#L2

```suggestion
      (errorKey: string, obj: ErrResponse) => {
```